content/collections/account-management/en/self-service-data-deletion-in-amplitude.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 1b48f869-0e38-45ac-a6d5-03184a2ac77d
+blueprint: account-management
+title: 'Self-service data deletion in Amplitude'
+this_article_will_help_you:
+  - 'Delete data from Amplitude permanently'
+landing: false
+exclude_from_sitemap: false
+updated_by: 5817a4fa-a771-417a-aa94-a0b1e7f55eae
+updated_at: 1728428160
+---
+Sometimes, you may need to permanently remove data from your Amplitude projects in a timely manner. For example, maybe your product sent incorrect data to Amplitude last month, but that data has since been corrected, and you’d like to remove the original, incorrect events or properties.
```

Comment:
```suggestion
Sometimes, you may need to permanently remove data from your Amplitude projects. For example, maybe your product sent incorrect data to Amplitude last month. That data has since been corrected, and you’d like to remove the incorrect events or properties.
```
Second sentence read kind of long. Took a stab at consollidationl